includes/admin/class-wp-job-manager-settings.php

```diff
@@ -552,13 +552,26 @@ public function output() {
 
 				<?php settings_fields( $this->settings_group ); ?>
 
-				<h2 class="nav-tab-wrapper">
-					<?php
-					foreach ( $this->settings as $key => $section ) {
-						echo '<a href="#settings-' . esc_attr( sanitize_title( $key ) ) . '" class="nav-tab">' . esc_html( $section[0] ) . '</a>';
-					}
-					?>
-				</h2>
+				<div class="job-manager-settings-header-wrap">
+					<div class="job-manager-settings-header">
+						<div class="job-manager-settings-header-row">
+							<img class="job-manager-settings-logo"
+								src="<?php echo esc_attr( JOB_MANAGER_PLUGIN_URL . '/assets/images/jm-full-logo.png' ); ?>"
+								alt="<?php esc_attr_e( 'Job Manager', 'wp-job-manager' ); ?>" />
+							<input type="submit" class="job-manager-settings-submit wpjm-button is-outline" value="<?php esc_attr_e( 'Save Changes', 'wp-job-manager' ); ?>" />
+						</div>
+
+						<div class="nav-tab-wrapper">
+							<?php
+							foreach ( $this->settings as $key => $section ) {
+								echo '<a href="#settings-' . esc_attr( sanitize_title( $key ) ) . '" class="nav-tab">' . esc_html( $section[0] ) . '</a>';
```

Comment:
`esc_url()` here, too.  Probably best to wrap the whole URL inside `esc_url()` though.  

Something like
```
echo '<a href="' . esc_url( '#settings-' . sanitize_title( $key ) ) . '" class="nav-tab">' . esc_html( $section[0] ) . '</a>';
```